.release-please-manifest.json

```diff
@@ -28,6 +28,7 @@
   "plugins/node": "2.0.0",
   "plugins/nodemon": "2.0.0",
   "plugins/npm": "2.0.1",
+  "plugins/pa11y": "6.2.3",
```

Comment:
**suggestion**: we want to release pa11y as version 0.1.0 instead of 6.2.3, so we should change this to 0.1.0